app/routes/upload.tsx

```
@@ -0,0 +1,137 @@
+
+    return (
+        <main className="bg-[url('/images/bg-main.svg')] bg-cover">
+            <Navbar />
+
+            <section className="main-section">
+                <div className="page-heading py-14">
+                    <h1>Smart feedback for your dream job</h1>
+                    {isProcessing ? (
+                        <>
+                            <h2>{statusText}</h2>
+                            <img src="/images/resume-scan.gif" className="w-full" />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)